### PR TITLE
Fix installation for Ubuntu in README.md

### PR DESCRIPTION
**Pull request by shaobosong**
- **PR Description**
If '/usr/local/bin' does not exist, 'install' will eventually result in a regular file named '/usr/local/bin' being created.

- **Please check if the PR fulfills these requirements**

* [ ] Cheatsheets are up-to-date (run `go generate ./...`)
* [ ] Code has been formatted (see [here](https://github.com/jesseduffield/lazygit/blob/master/CONTRIBUTING.md#code-formatting))
* [ ] Tests have been added/updated (see [here](https://github.com/jesseduffield/lazygit/blob/master/pkg/integration/README.md) for the integration test guide)
* [ ] Text is internationalised (see [here](https://github.com/jesseduffield/lazygit/blob/master/CONTRIBUTING.md#internationalisation))
* [ ] If a new UserConfig entry was added, make sure it can be hot-reloaded (see [here](https://github.com/jesseduffield/lazygit/blob/master/docs/dev/Codebase_Guide.md#using-userconfig))
* [ ] Docs have been updated if necessary
* [x] You've read through your own file changes for silly mistakes etc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -309,7 +309,7 @@ sudo eopkg install lazygit
 LAZYGIT_VERSION=$(curl -s "https://api.github.com/repos/jesseduffield/lazygit/releases/latest" | grep -Po '"tag_name": "v\K[^"]*')
 curl -Lo lazygit.tar.gz "https://github.com/jesseduffield/lazygit/releases/latest/download/lazygit_${LAZYGIT_VERSION}_Linux_x86_64.tar.gz"
 tar xf lazygit.tar.gz lazygit
-sudo install lazygit /usr/local/bin
+sudo mkdir -p /usr/local/bin/ && sudo install lazygit /usr/local/bin/
```
Comment (mark2185) [inline README.md:312]: I think a simpler form would be better:
```shell
sudo install lazygit -D -t /usr/local/bin
```
From `man install`:
```
-D     create all leading components of DEST except the last, or all components of --target-directory, then copy SOURCE to DEST
```

Comment (shaobosong) [reply README.md:312]: Thanks for your advice!

**Comment by mark2185** [comment]
LGTM

**Comment by mark2185** [comment]
If you could squash your commits, that would be great. There's no need for both of them.

**Pull request merged**